amd/src/package_search/components/tab_content.js

```diff
@@ -35,7 +36,16 @@ export default class extends Component {
         this.category = descriptor.category;
         this.selectors = {
             CONTENT: ".qpy-tab-content",
+            PAGINATION: '[data-for="pagination"]',
         };
+
+        // Register pagination.
+        new Pagination({
+            element: this.getElement(this.selectors.PAGINATION),
+            name: `pagiation_${this.category}`,
+            reactive: descriptor.reactive,
+            category: this.category,
+        });
```

Comment:
Ich denke nicht, dass das so einfach geht, da die Komponenten das `reactive`-Modul bei der Initialisierung benötigen.

Reply:
Die Komponenten können die reactive Instanz auch selber finden:

> In case no Reactive Instance is passed, the BaseComponent will try to inherit the instance from a parent DOM element. This way subcomponents can omit that param.

https://moodledev.io/docs/guides/javascript/reactive#instantiate-a-component

Reply:
Ah, das wusste ich noch nicht! Das würde ich in einem anderen PR machen und direkt alle Komponenten so initialisieren.